### PR TITLE
Document build acceleration log analysis

### PR DESCRIPTION
**Pull request by drewnoakes**
Provides guidance for verbose build acceleration logs, and tips for diagnosing problems.

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/project-system/pull/8802)

### CODE REVIEW

File: docs/build-acceleration.md [hunk 1]
```diff
@@ -49,6 +49,8 @@ You may disable build acceleration for specific projects in your solution by red
 
 ## Debugging
 
+### Enable logging
+
 Build acceleration runs with the FUTDC, and outputs details of its operation in the build log. To enable this logging:
 
 > Tools | Options | Projects and Solutions | .NET Core
```
Comment (tmeschter) [inline docs/build-acceleration.md:56]: I think you mean:

```suggestion
> Tools | Options | Projects and Solutions | SDK-Style Projects
```

File: docs/build-acceleration.md [hunk 2]
```diff
@@ -61,6 +63,73 @@ Setting _Logging Level_ to a value other than `None` results in messages prefixe
 - `Minimal` produces a single message per out-of-date project.
 - `Info` and `Verbose` provide increasingly detailed information about the inner workings of the check, which are useful for debugging.
 
+### Validate builds are accelerated
+
+If build acceleration cannot be enabled for any of the reasons given below, builds continue to work as before but will be slower.
```
Comment (tmeschter) [inline docs/build-acceleration.md:68]: I think you can just drop the "will be slower" part:

```suggestion
If build acceleration cannot be enabled for any of the reasons given below, builds continue to work as before.
```

They will be slower than if build acceleration could be enabled, but they won't be slower than they were before the build acceleration feature came along.

File: docs/build-acceleration.md [hunk 3]
```diff
@@ -61,6 +63,73 @@ Setting _Logging Level_ to a value other than `None` results in messages prefixe
...
+1. Modify source of a referenced project (either a direct reference or transitive reference).
+1. Build the project again.
+
+Looking through the build output with the following points in mind:
+
+- ℹ️ If you see:
+
+   > This project appears to be a candidate for build acceleration. To opt in, set the 'AccelerateBuildsInVisualStudio' MSBuild property to 'true'.
+
+   Then the project does not specify the `AccelerateBuildsInVisualStudio` property, or its value was not `true` or `false`, and the project like it would benefit from build acceleration. If the project cannot use build acceleration for any reason, this message can be suppressed by setting the property to `false` explicitly. See [configuration](#configuration) to learn how to configure build acceleration correctly.
```
Comment (tmeschter) [inline docs/build-acceleration.md:89]: Did you mean this?

```suggestion
   Then the project does not specify the `AccelerateBuildsInVisualStudio` property, or its value was not `true` or `false`, and the project likely would benefit from build acceleration. If the project cannot use build acceleration for any reason, this message can be suppressed by setting the property to `false` explicitly. See [configuration](#configuration) to learn how to configure build acceleration correctly.
```

**Review by tmeschter — APPROVED** [review]

File: docs/build-acceleration.md (hunk 1 above)
Comment (tmeschter) [reply docs/build-acceleration.md:56]: Also in the caption of the image below this line, but I couldn't leave a comment on that. 🤷🏻‍♂️

**Review by melytc — APPROVED** [review]

File: docs/build-acceleration.md [hunk 4]
```diff
@@ -61,6 +63,73 @@ Setting _Logging Level_ to a value other than `None` results in messages prefixe
...
 - `Info` and `Verbose` provide increasingly detailed information about the inner workings of the check, which are useful for debugging.
 
+### Validate builds are accelerated
+
+If build acceleration cannot be enabled for any of the reasons given below, builds continue to work as before but will be slower.
+
+The following prerequisites exist for build acceleration:
+
+- You are running Visual Studio 2022 version 17.5 or later.
+- The project is an SDK-style .NET project.
```
Comment (MiYanni) [inline docs/build-acceleration.md:73]: Have we tested this for VB/F#?

**Review by MiYanni — APPROVED** [review]

File: docs/build-acceleration.md (hunk 1 above)
Comment (drewnoakes) [reply docs/build-acceleration.md:56]: Good catch. This was copied from the FUTDC document, so I'll update it there too.

File: docs/build-acceleration.md (hunk 4 above)
Comment (drewnoakes) [reply docs/build-acceleration.md:73]: VB: yes, because we are dogfooding it in this repo.
F#: not yet. I'll make sure that happens.

**Pull request merged**